specs/social.md

```diff
@@ -0,0 +1,30 @@
+# DID format
+
+<dl>
+	<dt>Namespace</dt><dd>`social`</dd>
+	<dt>Version</dt><dd>1.0</dd>
+</dl>
+
+This is an application format for describing how the communal social Namespace works.
+
+## Encryption
+
+Clients MUST respect the [encryption](encryption.md) format. Everything that follows is assumed to be after that's decrypted things as appropriate.
+
+## Document prefix
+
+Everything is prefixed using the [did format](did.md), followed by `social/1.0/`
+
+e.g. `/did/1.0/did:key:z6MkhaXgBZDvotDkL5257faiztiGiC2QtKLGpbnnEGta2doK/social/1.0/...`
+
+## DID level contents
+
+`about.yaml` - follows the [about spec](about.md)
+
+`feed/` - follows the [activities spec](activities.md). This is where you post your general social media... "stuff".
+- Suggestion (only) - use `feed/public` and `feed/followers` with different capabilities set, so only followers can see things under that section.
+- Suggestion (only) - use date-based paths for nice urls - e.g. `feed/public/2024/04/09/1711062719000.yaml`
+- e.g. `/did/1.0/did:key:z6MkhaXgBZDvotDkL5257faiztiGiC2QtKLGpbnnEGta2doK/social/1.0/feed/public/2024/04/09/1711062719000.yaml`
+
+`followers/` - follows the [trusted-dids spec](trusted-dids.md).
```

Comment:
This can be exposed publicly or only to your followers, at your discretion

Reply:
Should this be `following`?

Reply:
> This can be exposed publicly or only to your followers, at your discretion

Oh, I covered that here: https://github.com/activitystar/activitystar.github.io/pull/1/files#diff-4ff4dc3d1d42bb01d401ea8a7adb534cb345db8d4d9222c903fa7eaa3ca265b5R22